main.go

```diff
@@ -136,16 +214,42 @@ func (c *graphiteCollector) processLine(line string) {
 	}
 	originalName := parts[0]
 	var name string
-	mapping, labels, present := c.mapper.GetMapping(originalName, mapper.MetricTypeGauge)
+	var err error
+	mapping, labels, mappingPresent := c.mapper.GetMapping(originalName, mapper.MetricTypeGauge)
```

Comment:
The mapping applies to the metric name, so I think we need to extract the Graphite tags before attempting GetMapping.

Reply:
Done